include/crocoddyl/core/actions/diff-lqr.hxx

```diff
@@ -59,6 +59,22 @@ void DifferentialActionModelLQRTpl<Scalar>::calc(const boost::shared_ptr<Differe
       Scalar(0.5) * x.dot(Lxx_ * x) + Scalar(0.5) * u.dot(Luu_ * u) + x.dot(Lxu_ * u) + lx_.dot(x) + lu_.dot(u);
 }
 
+template <typename Scalar>
+void DifferentialActionModelLQRTpl<Scalar>::calc(const boost::shared_ptr<DifferentialActionDataAbstract>& data,
+                                                 const Eigen::Ref<const VectorXs>& x) {
+  if (static_cast<std::size_t>(x.size()) != state_->get_nx()) {
+    throw_pretty("Invalid argument: "
+                 << "x has wrong dimension (it should be " + std::to_string(state_->get_nx()) + ")");
+  }
+
+  if (drift_free_) {
+    data->xout.setZero();
+  } else {
+    data->xout = f0_;
```

Comment:
Is it really necessary to update `xout`? Is it going to be used?

Reply:
No, we don't have to. Apologies, I forgot to modify this bit.
Now it is done!